docs/GetStarted/getting_started_linux_cmake.md

```diff
@@ -16,36 +16,40 @@ documented separately, as they require further setup.
 
 ## Prerequisites
 
-### Install CMake
+### Generic and IDE setup
 
-Install CMake version >= 3.13:
+See shortcuts for specific distros below.
 
-```shell
-$ sudo apt install cmake
-```
+#### Install CMake
+
+* Install CMake version >= 3.13
 
 > Tip:<br>
 > &nbsp;&nbsp;&nbsp;&nbsp;Your editor of choice likely has plugins for CMake,
 > such as the Visual Studio Code
 > [CMake Tools](https://github.com/microsoft/vscode-cmake-tools) extension.
 
-### Install Ninja
+#### Install Ninja
 
 [Ninja](https://ninja-build.org/) is a fast build system that you can use as a
 CMake generator. Follow Ninja's
 [installing documentation](https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages).
 
-### Install a Compiler
+#### Install a Compiler
 
 We recommend Clang. GCC is not fully supported.
 
+Set environment variables:
+
 ```shell
-$ sudo apt install clang
+export CC=clang
+export CXX=clang++
 ```
 
-Set environment variables:
+#### Short-cuts for Debian (>= Buster)/Ubuntu
 
 ```shell
+sudo apt install cmake ninja-build clang python3-dev libvulkan-dev
```

Comment:
I would prefer to keep Python and Vulkan isolated to their own pages, though including here is pretty harmless and saves a bit of time if you're going through the docs in sequence. Up to you if you want to keep them here.